python-cffi/netplan/netdef.py

```diff
@@ -388,7 +390,8 @@ def __next__(self):
             'mtubytes': next_value.mtubytes,
             'congestion_window': next_value.congestion_window,
             'advertised_receive_window': next_value.advertised_receive_window,
-            'onlink': next_value.onlink
+            'onlink': next_value.onlink,
+            'advertised_mss': next_value.advmss
```

Comment:
I'm not sure about this one. But considering this is part of a private data structure `_NetdefRouteIterator`, accessing internal data structures without a getter should be acceptable.